inform/story.ni

```diff
@@ -82,7 +82,7 @@ Chapter 2 - Yard
 
 Section 1 - Out Front
 
-Front Yard is west of Front Porch.
+Front Yard is west of Front Porch and northeast of 21stBayview.
```

Comment:
The in-game nav works fine, it's just the description strings that we need to change.